rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxSeekBar.java

```diff
@@ -16,7 +16,33 @@
    */
   @CheckResult @NonNull
   public static Observable<Integer> changes(@NonNull SeekBar view) {
-    return Observable.create(new SeekBarChangeOnSubscribe(view));
+    return Observable.create(new SeekBarChangeOnSubscribe(view, null));
+  }
+
+  /**
+   * Create an observable of progress value changes on {@code view} that were made only from the user.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Note:</em> A value will be emitted immediately on subscribe.
+   */
+  @CheckResult @NonNull
+  public static Observable<Integer> changesOnlyFromUser(@NonNull SeekBar view) {
```

Comment:
I think we should call this `userChanges` to match what's described in the README for differentiation.